google/cloud/storage/blob.py

```diff
@@ -3018,6 +3019,18 @@ def custom_time(self):
         if value is not None:
             return _rfc3339_to_datetime(value)
 
+    @custom_time.setter
+    def custom_time(self, value):
+        """Set the custom time for the object.
+
+        See https://cloud.google.com/storage/docs/json_api/v1/objects
+
+        :type value: :class:`datetime.datetime`
+        :param value: (Optional) Set the custom time of blob. Datetime object parsed
```

Comment:
Document the semantics of `None` value (which should be to clear any existing property value).